### PR TITLE
git-clone: add core.sshCommand example

### PR DESCRIPTION
**Pull request by navarroaxel**
- [ ] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by bl-ue — APPROVED** [review]
👍🏻

File: pages/common/git-clone.md [hunk 1]
```diff
@@ -30,3 +30,7 @@
 - Clone an existing repository only fetching a specific branch:
 
 `git clone --branch {{name}} --single-branch {{remote_repository_location}}`
+
+- Clone an existing repository using a specific SSH command:
+
+`git clone --config core.sshCommand="{{ssh -i path/to/private_ssh_key}}" {{remote_repository_location}`
```
Comment (marchersimon) [inline pages/common/git-clone.md:36]:
```suggestion
`git clone --config core.sshCommand="{{ssh -i path/to/private_ssh_key}}" {{remote_repository_location}}`
```

**Review by CleanMachine1 — APPROVED** [review]
:shipit:

File: pages/common/git-clone.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/common/git-clone.md]: Does the bot not spot these?

Comment (bl-ue) [reply pages/common/git-clone.md]: No, it doesn't. https://github.com/tldr-pages/tldr-lint/issues/76

**Review by marchersimon — APPROVED** [review]
👍🏽

**Pull request merged**